website/content/docs/nia/terraform-modules.mdx

```diff
@@ -33,11 +33,11 @@ See below sections for more information on [defining source input](#source-input
 
 CTS monitors defined objects to provide source input to a task. This source input satisfies the requirements defined by the task's Terraform Module's [input variables](https://www.terraform.io/docs/language/values/variables.html).
 
-A task's source input slightly defers the task's condition even though both monitor defined objects. The task's condition monitors defined objects with a configured criteria. When this criteria is satisfied, the task will trigger.
+A task's source input is slightly different from the task's condition, even though both monitor defined objects. The task's condition monitors defined objects with a configured criteria. When this criteria is satisfied, the task will trigger.
 
-The source input however, monitors defined objects with the intent of providing values or metadata about these objects to the Terraform Module. The monitored source input and condition objects can be the same object, such as a task is configured with a `condition "services"` block and `source_includes_var` is set to `true`. The source input and condition can be different objects and configured separately, such as a task configured with a `condition "catalog-services` and `source_input "consul-kv"` block. In this way the source input does not need to be tied to the provided condition in order to satisfy the Terraform Module.
+The source input however, monitors defined objects with the intent of providing values or metadata about these objects to the Terraform Module. The monitored source input and condition objects can be the same object, such as a task is configured with a `condition "services"` block and `source_includes_var` is set to `true`. The source input and condition can be different objects and configured separately, such as a task configured with a `condition "catalog-services` and `source_input "consul-kv"` block. This results in the monitored source input not needing to be tied to the provided condition in order to satisfy the Terraform Module.
 
-Each type of monitored object can only be defined through one configuration means for a task. For example, if a task monitors services, the task cannot have both `condition "services"` and `source_input "services"` configured. See [Task Source Input configuration](/docs/nia/configuration#task-source-input) for more details.
+Each type of monitored object can only be defined through one configuration definiton of a task. For example, if a task monitors services, the task cannot have both `condition "services"` and `source_input "services"` configured. See [Task Source Input configuration](/docs/nia/configuration#task-source-input) for more details.
```

Comment:
```suggestion
Each type object that CTS monitors can only be defined through one task definition configuration. For example, if a task monitors services, the task cannot have both `condition "services"` and `source_input "services"` configured. See [Task Source Input configuration](/docs/nia/configuration#task-source-input) for more details.
```
Is that clearer/correct?

Reply:
Ah, it's made me realize how unclear I was. I was Maybe:
```
Each type object that CTS monitors can only be defined through one configuration within a task definition.
```